web/src/router.rs

```diff
@@ -264,36 +302,44 @@ mod organization_endpoints_tests {
             .client
             .get(test_client_server.url("/organizations").unwrap())
             .send()
-            .await?
-            .text()
             .await?;
 
-        let organization1 = &organizations[0][0];
-        let organization2 = &organizations[0][1];
-        let organization3 = &organizations[0][2];
+        // We need to parse the values to serde_json::Value to compare them
+        // // so that the attribute order does not matter.
+        let parsed_response: serde_json::Value =
+            serde_json::from_str(&response.text().await?).unwrap();
+        let organizations: serde_json::Value = json!(&organizations);
 
-        assert_eq!(
-            response,
-            json!([organization1, organization2, organization3]).to_string()
-        );
+        assert_eq!(parsed_response, organizations[0]);
 
         Ok(())
     }
 
-    // Purpose: adds multiple Organization instances to a mock DB and tests that calling
-    // the appropriate endpoint deletes instances specified by distinct IDs.
+    // // Purpose: adds multiple Organization instances to a mock DB and tests that calling
+    // // the appropriate endpoint deletes instances specified by distinct IDs.
```

Comment:
```suggestion
    // the appropriate endpoint deletes instances specified by distinct IDs.
```

Reply:
updated in 3d201b56bb1b97fe95447580f25b3ce9ba5afb7f!